### PR TITLE
fixes typo in ro sign

### PR DESCRIPTION
**Pull request by AndroBetel**
# About the pull request

fixes a typo in req rules sign

# Explain why it's good for the game

typo fix
also replaces deleted RO with QM

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

cyrosleep => cryosleep
</details>

# Changelog

:cl:
spellcheck: Fixed a typo in RO rules sign.
spellcheck: RO mention is no more, replaced with QM.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/signs.dm [hunk 1]
```diff
@@ -570,7 +570,7 @@
 
 /obj/structure/sign/ROsign
 	name = "\improper USCM Requisitions Office Guidelines"
-	desc = " 1. You are not entitled to service or equipment. Attachments are a privilege, not a right.\n 2. You must be fully dressed to obtain service. Cyrosleep underwear is non-permissible.\n 3. The Requsitions Officer has the final say and the right to decline service. Only the Acting Commanding Officer may override their decisions.\n 4. Please treat your Requsitions staff with respect. They work hard."
+	desc = " 1. You are not entitled to service or equipment. Attachments are a privilege, not a right.\n 2. You must be fully dressed to obtain service. Cryosleep underwear is non-permissible.\n 3. The Requsitions Officer has the final say and the right to decline service. Only the Acting Commanding Officer may override their decisions.\n 4. Please treat your Requsitions staff with respect. They work hard."
```
Comment (Birdtalon) [inline code/game/objects/structures/signs.dm:573]: The Requisitions officer no longer exists having been replaced with the Axillary Support Officer so might as well change that here too.

**Comment by fira** [comment]
I can't believe nobody ever noticed it

File: code/game/objects/structures/signs.dm (hunk 1 above)
Comment (kiVts) [reply code/game/objects/structures/signs.dm:573]: What, this is nonsense

Comment (AndroBetel) [reply code/game/objects/structures/signs.dm]: changed to Quartermaster

**Review by fira — APPROVED** [review]

**Pull request merged**